packages/quick_actions/quick_actions_android/android/src/main/java/io/flutter/plugins/quickactions/MethodCallHandlerImpl.java

```diff
@@ -54,13 +55,12 @@ public void onMethodCall(MethodCall call, MethodChannel.Result result) {
     switch (call.method) {
       case "setShortcutItems":
         if (Build.VERSION.SDK_INT > Build.VERSION_CODES.N_MR1) {
-          List<Map<String, String>> serializedShortcuts = call.arguments();
+          List<Map<String, String>> serializedShortcuts = Objects.requireNonNull(call.arguments());
           List<ShortcutInfo> shortcuts = deserializeShortcuts(serializedShortcuts);
 
           Executor uiThreadExecutor = new UiThreadExecutor();
           ThreadPoolExecutor executor =
-              new ThreadPoolExecutor(
-                  0, 1, 1, TimeUnit.SECONDS, new LinkedBlockingQueue<Runnable>());
+              new ThreadPoolExecutor(0, 1, 1, TimeUnit.SECONDS, new LinkedBlockingQueue<>());
```

Comment:
What's the warning with `Runnable`? Just curious

Reply:
I'm pretty sure this is one of the AS-specific lints rather than something that shows up in our tooling checks, but I cleaned it up since it's trivial and reduces warning noise in Android Studio. The message is just something along the lines of "LinkedBlockingQueue<Runnable> could be LinkedBlockingQueue<>". It's because specifying the generic isn't necessary when it can be clearly inferred from context (like here, where the method parameter requires a `...<Runnable>`).